packages/prettier-config/README.md

```diff
@@ -0,0 +1,76 @@
+# `@shopify/prettier-config`
+
+[![Build Status](https://travis-ci.com/Shopify/web-foundation.svg?branch=master)](https://travis-ci.com/Shopify/web-foundation)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](../../LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Fprettier-config.svg)](https://badge.fury.io/js/%40shopify%2Fprettier-config.svg)  [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/prettier-config.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/prettier-config.svg) 
+
+Shared prettier configuration
+
+## Installation
+
+```bash
+$ yarn add --dev @shopify/prettier-config
+```
+
+## Usage
+Shopify’s shared prettier config comes bundled in `@shopify/prettier-config`. To enable these rules, add a `prettier` property in your `package.json` and reference this shared config as follows:
+```
+"prettier": "@shopify/prettier-config"
+```
+Previously, rules had been defined directly in the `package.json` 
+```
+"prettier": {
+  "arrowParens": "avoid",
+  "singleQuote": true,
+  "bracketSpacing": false,
+  "trailingComma": "all",
+  "quoteProps": "as-needed"
+}
+```
+or in a `.prettierrc` resulting in duplication across projects
+```
+{
+  "arrowParens": "avoid",
+  "singleQuote": true,
+  "bracketSpacing": false,
+  "trailingComma": "all",
+  "quoteProps": "as-needed"
+}
+```
+Any previous .prettierrc should be removed in favour of the shared config.
+See [configuring prettier](https://prettier.io/docs/en/configuration.html) and [sharing configuration](https://prettier.io/docs/en/configuration.html#sharing-configurations) for more details.  
+
+This shared prettier config integrates with [@shopify/eslint-plugin](https://github.com/Shopify/web-foundation/blob/master/packages/eslint-plugin/README.md) and [@shopify/stylelint-plugin](https://github.com/Shopify/web-foundation/blob/master/packages/stylelint-plugin/README.md).
+
+The following is an example configuration using `eslint-plugin`, `stylelint-plugin` and shared `prettier-config`:
```

Comment:
I think we can omit this section and code snippet. If folks are interested, they can explore the code. Large snippets like this can get stale.